schema/schema.json

```diff
@@ -139,6 +139,31 @@
                 "noNewPrivileges": {
                     "id": "https://opencontainers.org/schema/bundle/process/linux/noNewPrivileges",
                     "type": "boolean"
+                },
+                "rlimits": {
+                    "id": "https://opencontainers.org/schema/bundle/linux/rlimits",
+                    "type": "array",
+                    "items": [
+                        {
```

Comment:
fair. though it is not strict. I can change it, but it works as is.

On Wed, Apr 6, 2016 at 5:30 PM, W. Trevor King <notifications@github.com>
wrote:

> In schema/schema.json
> <https://github.com/opencontainers/runtime-spec/pull/370#discussion_r58785573>
> :
>
> > @@ -139,6 +139,31 @@
> >                  "noNewPrivileges": {
> >                      "id": "https://opencontainers.org/schema/bundle/process/linux/noNewPrivileges",
> >                      "type": "boolean"
> > +                },
> > +                "rlimits": {
> > +                    "id": "https://opencontainers.org/schema/bundle/linux/rlimits",
> > +                    "type": "array",
> > +                    "items": [
> > +                        {
>
> JSON Schema's items takes an object, not an array of objects. The docs
> are pretty sparse here, but see this example
> <https://tools.ietf.org/html/draft-zyp-json-schema-04#section-7.2.3>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/370/files/fc6670d4ae475faa20196488c87c373208a09223#r58785573>
>